byway/src/main/webapp/routepage.js

```diff
@@ -49,27 +64,135 @@ function calcRoute(directionsService, directionsRenderer, start, end) {
   });
 }
 
+/** Get the new list of stops from datastore onload */
+function getStops(){
+  const stopList = document.getElementById('stop-list');
+  if(stopList != null){
+        stopList.innerText = ""; // clear list
+    }
+  fetch('/api/stop')
+  .then(response => response.json())
+  .then((stops) => {
+    stops.forEach((stop)=>{
+      var btn = document.createElement('button');
+      btn.id = stop.id;
+      btn.innerText = stop.placename;
+      btn.setAttribute("class", "btn rec-btn");
```

Comment:
Consider using `btn.className` instead of setting the attribute. I was told by Zach we should prefer accessing attributes this way instead. Also, you might want to use `btn.classList.add("btn rec-btn")` instead to append classes to the button in case someone would want to add more to it. Might not be a big issue since you initialize said button here, but note the use of `classList.add()` and `classList.remove()` in case you want to modify a class.